Source/AliveLibAE/Renderer/IRenderer.hpp

```diff
@@ -59,6 +62,8 @@ class IRenderer
 
     virtual void Upload(BitDepth bitDepth, const PSX_RECT& rect, const u8* pPixels) = 0;
 
+    virtual void LoadCustomCAM(const char* path, const unsigned char* key, int keyLength) = 0;
```

Comment:
shouldn't "Custom" really be "External" or something "custom" and "data" is super generic name